packages/api-client/src/views/Request/libs/oauth2.ts

```diff
@@ -52,9 +55,25 @@ export const authorizeOauth2 = (
       else if (scheme.flow.type === 'authorizationCode')
         url.searchParams.set('response_type', 'code')
 
+      // Handle relative redirect uris
+      if (scheme.flow['x-scalar-redirect-uri'].startsWith('/')) {
```

Comment:
Let me add docs for x-scalar-redirect-uri tomorrow ✌️

Reply:
Why do we need it?

Reply:
why do we need to check for the `/` or why do we need x-scalar-redirect-uri?

Reply:
if its the latter, lets discuss

Reply:
The latter, why do we need the custom attribute and can’t just use the existing redirectUri attribute?

Reply:
Can tag @geoffgscott but this is something I would like to discuss in person